src/Domain/OutputType.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace PhpTuf\ComposerStager\Domain;
+
+/**
+ * Process output type constants.
+ *
+ * @see \Symfony\Component\Process\Process::readPipes
+ */
+final class OutputType
```

Comment:
I can't see where we use this except for comment examples. We reference this in the @param $type doc for `\PhpTuf\ComposerStager\Console\Output\ProcessCallback::__invoke()` but I am not sure we use that param

I checked and deleting file doesn't cause any test failures

Reply:
_We_ don't use this param because we never invoke the callbacks--Symfony Process does. Our console command prints all output without distinguishing between standard and error output. But clients may want to distinguish, and I don't want them depending on a third party library, so I'm just giving them a stable facade of sorts. Adam asked this same question, though... maybe I'll add some documentation.

Reply:
ok thanks for the explaining. Since `symfony/process` is direct dependency of this library would it really be so bad if they callers depended on it. Can you imagine we might switch to something else with major version jump?